### DISCUSSION TITLE
Vercel-Timeout-error

### DISCUSSION THREAD

**Discussion by SySagar** [vercel/community#2852] [Help] [answered]
### Summary

I tried to push a nextauth secret provider to vercel and added them to environment variables in vercel. However I am consistenly now getting this error.
![Capture](https://github.com/vercel/community/assets/93787385/88b553b9-dbd4-41e2-97c1-32e610205809)

p.s. - I accidentally committed .env in git. In order to remove it I ran this command 

`git filter-branch --index-filter 'git rm -rf --cached --ignore-unmatch .env HEAD`

then I forced push. Thereafter it shows error like this

### Steps to Reproduce

*

**Comment by swarnava** [comment, marked as answer]
Hi there,

Thanks for reaching out to Vercel Support!

At this time we are currently experiencing issues deploying Serverless Functions to the `iad1` region, this results from an upstream provider. Please try and update your Serverless Function region to `cle1` as a workaround and redeploy: https://vercel.com/docs/concepts/functions/serverless-functions/regions. We have updated our Status Page with additional information. https://www.vercel-status.com/incidents/cqkdqrzqkspm

**Comment by SySagar** [comment]
Thanks! This helped me a lot.
